### PR TITLE
Integrating redis with app

### PR DESCRIPTION
**Pull request by MeenuV**

### CODE REVIEW

**Comment by clever-alice** [comment]
your branch seems to be failing tests (not sure why the build is passing). I think you forgot to put redis in requirements?

File: client/scraper/metadata.py [hunk 1]
```diff
@@ -73,6 +75,8 @@ def parse_content(self, response):
 
     def to_json(self):
         json_data = json.dumps(self.prop_map)
+        if config.CACHE_DATA:
```
Comment (clever-alice) [inline client/scraper/metadata.py]: I don't think cache should be in `to_json`. We should instead put it in a new function like `cache_json_data` and call it in `api_handler`

**Comment by clever-alice** [comment]
Make sure you test for both CACHE = True & False, getting some errors when running

**Comment by AliceYuan** [comment]
Also add redis-server command to `build.sh` file

File: client/scraper/metadata.py (hunk 1 above)
Comment (MeenuV) [inline client/scraper/metadata.py]: Makes sense. Will change

**Comment by AliceYuan** [comment]
:+1:

**Pull request merged**